app/src/main/kotlin/net/corda/cli/application/logger/LoggerStream.kt

```diff
@@ -0,0 +1,64 @@
+package net.corda.cli.application.logger
+
+import java.io.OutputStream
+import java.io.PrintStream
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.io.ByteArrayOutputStream
+
+enum class LogLevel {
+    INFO, ERROR
+}
+
+class LoggerStream(
```

Comment:
```suggestion
internal class LoggerStream(
```

also I would move `sysOut` and `errOut` up and outside of the class such that they will be accessible by plugins as: `net.corda.cli.application.logger.sysOut`.

Reply:
they wouldn't be accessible unless they were injected into the plugins via interface in the api module, however they can call LoggerFactory.getLogger('systemOut') etc once the plugin has started.

Reply:
Ah, yes - I have forgotten about `api` aspect.
IMO accessing them using `LoggerFactory.getLogger('systemOut')` is fine.